src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -8,7 +8,7 @@ <h1>Home</h1>
 </div>
 
 <div class="row mb-3" *tmIsLoading="!hasCoursesLoaded">
-  <div class="col-4">
+  <div class="col-4" style="margin-top: auto;">
```

Comment:
Let's try to use the css classes provided by bootstrap, instead of inline styling.